src/net/Lib/Images/DefaultImageProvider.cs

```diff
@@ -0,0 +1,19 @@
+using System;
```

Comment:
Please put in /Providers folder to match the other Providers, Data/Messaging....also put in Providers namespaces